orchagent/port.h

```diff
@@ -55,6 +57,8 @@ class Port
     int                 m_ifindex = 0;
     sai_object_id_t     m_port_id = 0;
     sai_object_id_t     m_vlan_oid = 0;
+    sai_object_id_t     m_bridge_id = 0;
```

Comment:
not exactly, there is .1q bridge port which is 1:1 mapping to the physical port/lag. there is .1d bridge which is (port, vlan) tuple.

---
In reply to: [126781483](https://github.com/Azure/sonic-swss/pull/250#discussion_r126781483) [](ancestors = 126781483)